src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_params.py

```diff
@@ -51,15 +64,15 @@ def __init__(self, value):
 register_cli_argument('redis', 'reboot_type', **enum_choice_list(RebootType))
 register_cli_argument('redis', 'key_type', **enum_choice_list(RedisKeyType))
 register_cli_argument('redis', 'shard_id', type=int)
+register_cli_argument('redis', 'sku', **enum_choice_list(SkuName))
+register_cli_argument('redis', 'vm_size', **enum_choice_list(VmSize))
+register_cli_argument('redis', 'enable_non_ssl_port', action='store_true')
+register_cli_argument('redis', 'shard_count', type=int)
+register_cli_argument('redis', 'subnet_id') # TODO: Create generic id completer similar to name
```

Comment:
Actually, pylint doesn't care. It's PEP8 that hates this... ... ... for some reason.